app/views/membership.html

```diff
@@ -0,0 +1,248 @@
+<project-header class="top-header"></project-header>
+<project-page class="membership" ng-if="project">
+  <div class="middle-section">
+    <div class="middle-container">
+      <div class="middle-header">
+        <div class="container-fluid">
+          <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
+        </div>
+      </div>
+      <div class="middle-content" persist-tab-state>
+        <div class="container-fluid">
+          <div class="row">
+            <div class="col-md-12">
+              <h1>
+                Membership
+                <small class="pull-right meta">
+                  <a target="_blank" href="{{'authorization' | helpLink}}">
+                    Learn more <i class="fa fa-external-link"></i>
```

Comment:
I used the `#roles` link here: 

![screen shot 2016-10-11 at 10 01 19 am](https://cloud.githubusercontent.com/assets/280512/19273222/adb35fc4-8f99-11e6-9b6f-e187be59d81b.png)

Shall we use the same anchor for both? I think that could make sense.